cli/waiter/token_post.py

```diff
@@ -183,10 +201,12 @@ def add_implicit_arguments(unknown_args, parser):
     for i in range(num_unknown_args):
         arg = unknown_args[i]
         if arg.startswith(("-", "--")):
-            if arg.endswith('-secs') or arg.endswith('-mins') or arg.endswith('-instances'):
-                arg_type = int
-            elif arg.endswith('-factor'):
-                arg_type = float
+            if arg.endswith('-secs') or arg.endswith('-mins') or arg.endswith('-instances') or \
+                    arg.endswith('-index') or arg.endswith('-level') or \
+                    arg.endswith('-failures') or arg.endswith('-length'):
```

Comment:
I find the following easier to read:
`any(arg.endswith(suffix) for suffix in ['-failures', '-index', '-instances', '-level', '-length', '-mins', '-secs'])`

Reply:
Agreed, done